recipes/hydra-zen/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "hydra-zen" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: e27979c16505a654918c8c36cc190cc61ded14054a4d84efd65c0b83e7078050
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=35.0.2
```

Comment:
It looks like https://github.com/mit-ll-responsible-ai/hydra-zen/blob/v0.12.1/pyproject.toml#L5
will imply https://github.com/pypa/setuptools_scm/blob/v7.0.5/setup.py#L53-L55
such that this lower bound should be lifted to `42`